content/docs/csidriver/installation/helm/powerflex.md

```diff
@@ -182,7 +182,9 @@ kubectl create -f deploy/kubernetes/snapshot-controller
     format and replace/update the secret.  
     - "insecure" parameter has been changed to "skipCertificateValidation" as insecure is deprecated and will be removed from use in config.yaml or secret.yaml in a future release. Users can continue to use any one of "insecure" or "skipCertificateValidation" for now. The driver would return an error if both parameters are used.
     - Please note that log configuration parameters from v1.5 will no longer work in v2.0 and higher. Please refer to the [Dynamic Logging Configuration](../../../features/powerflex#dynamic-logging-configuration) section in Features for more information.
-    
+    - If the user is using complex K8s version like "v1.21.3-mirantis-1", use below kubeVersion check in helm/csi-unity/Chart.yaml file.
```

Comment:
Shouldn't it be in known bug ? or have link to the known bug ?

Reply:
Yes this is ticket https://jira.cec.lab.emc.com/browse/KRV-2959 and  taken care in PR https://github.com/dell/csi-unity/pull/68/ . So u r suggesting to put in some other page in Documentation? @coulof

Reply:
@coulof - Are you asking if there is a known issue around this from helm side? or from our driver's side? Recently, the PowerFlex CSI driver team also ran into this and upon investigation found that Helm does not really provide a way to match regex. Now, we could fix it with appending a -0 to the end of the K8S version check, but that will open up possibilities of installing on non standard versions such as -rc0 etc, which we want to to avoid (essentially that check opens up floodgates to a lot of possibilities).

Reply:
Yes, to me it was more of a known bug and it seems it is present in the troubleshooting section here : https://github.com/dell/csm-docs/pull/170/files#diff-063c077e1f7bc20266026d2fe359c3a3caf411e1d742991e8fb969561ad45d05R21

So I'm all good now.

Note that I opened a bug for a similar issue (KRV-3249).